hpx/runtime/resource/detail/create_partitioner.hpp

```diff
@@ -47,13 +47,14 @@ namespace hpx { namespace resource { namespace detail
         int argc, char** argv, std::vector<std::string> ini_config,
         resource::partitioner_mode rpmode = resource::mode_default,
         runtime_mode mode = runtime_mode_default,
-        bool check = true);
+        bool check = true, int* result = nullptr);
```

Comment:
Am I missing something, or is `int *` basically used as an `optional<int>`?

Reply:
I used `int*` with a default value to avoid having to change any of the existing code relying on this function. It's not really (conceptually) a `optional<int>`

Reply:
Ok, fair enough.